frontend/src/app/modules/posts/components/edit-post/edit-post.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="-mb-[24px]">
+<div class="flex">
+  <div class="flex flex-row justify-end w-full">
+    <span class="material-symbols-outlined cursor-pointer" (click)="closeForm()">close</span>
+    <span *ngIf="!isLoading()" class="material-symbols-outlined cursor-pointer" (click)="onFormSubmit()">done</span>
+    <div *ngIf="isLoading()" class="w-[24px] h-[24px]">
+    <span class="loading loading-spinner loading-sm mt-[2px] ml-[2px]" (click)="onFormSubmit()"></span>
```

Comment:
w sumie nie wiem czy tu ten (click) powinien być

Reply:
Hmmm a jak inaczej to widzisz?

Reply:
W sensie na loading spinerze niepotrzebny nie?